client/asset/eth/cmd/getgas/README.md

```diff
@@ -0,0 +1,39 @@
+## Gas Estimator Utility
+
+`getgas` is a program for getting gas estimates for DEX-critical operations
+on Ethereum. Use `getgas` to generate gas tables (`dexeth.Gases`) when adding a
+new token or a new contract version.
```

Comment:
This suggests the gases from this tool can be inserted directly with no additional buffering or even rounding up.  Maybe this should say "Use `getgas` results as a basis for the gas tables when..."

We've targeted an actual observed gas use of ~75% of the gas cap.  For example, to get a `v0Gases` for a new L1 or the `Gases` in the `Tokens` map for a token, you'd take the gasses from this tool and multiply by something like 1.3 and use those values.